.gitignore

```diff
@@ -1,4 +1,5 @@
 build
-composer.lock
```

Comment:
composer.lock should not be committed, it's a library

Reply:
Thank you @falkenhawk !